src/comps/ui/AlertDisplay.tsx

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from "react";
 import Alert from "@mui/material/Alert";
-import DisplayLinks from "./DisplayLinks";
+import TextLinks from "./TextLinks";
```

Comment:
I suppose you could have simply kept the component called DisplayLinks to reduce the number of breaking changes, but this is fine